### PR TITLE
2 3 sample update to latest

### PR DESCRIPTION
**Pull request by aremo-ms**
Updated version of 2-3 Multi-Tenant sample with support of new Code Gen

### CODE REVIEW

File: 2-WebApp-graph-user/2-3-Multi-Tenant/README.md [hunk 1]
```diff
@@ -1,52 +1,51 @@
...
-This sample shows how to build an ASP.NET Core MVC web application that uses OpenID Connect to sign in users from multiple Azure AD tenants. Additionally it also introduces developers to the concept of a [multi-tenant](https://docs.microsoft.com/azure/active-directory/develop/single-and-multi-tenant-apps) Azure Active Directory application.
+This sample demonstrates a ASP.NET Core Web App calling Microsoft Graph.
+
+> :information_source: To learn how to integrate an application with Azure AD as a [multi-tenant](https://aka.ms/multi-tenant) app, consider going through the recorded session:[Develop multi-tenant applications with the Microsoft identity platform](https://www.youtube.com/watch?v=B416AxHoMJ4).
+
+> :information_source: To learn how applications integrate with [Microsoft Graph](https://aka.ms/graph), consider going through the recorded session:: [An introduction to Microsoft Graph for developers](https://www.youtube.com/watch?v=EBbnpFdB92A)
+
+## Scenario
 
 ### Overview
```
Comment (salman90) [inline 2-WebApp-graph-user/2-3-Multi-Tenant/README.md:50]: Do you need both overviews?

**Review by derisen — APPROVED** [review]
LGTM!

File: 2-WebApp-graph-user/2-3-Multi-Tenant/AppCreationScripts/sample.json [hunk 2]
```diff
@@ -31,7 +96,8 @@
           "Resource": "Microsoft Graph",
           "DelegatedPermissions": [ "User.Read.All" ]
         }
-      ]
+      ],
```
[the remark this answers is not in the document]
Comment (aremo-ms) [reply 2-WebApp-graph-user/2-3-Multi-Tenant/AppCreationScripts/sample.json:99]: the powershell should de it automatically

**Comment by kalyankrishna1** [comment]
1. All samples signin-in users should have 'acct' claim
1. Declare client capability and handle CAE events if calling MS Graph

**Review by kalyankrishna1 — APPROVED** [review]
:shipit:

File: 2-WebApp-graph-user/2-3-Multi-Tenant/README.md (hunk 1 above)
Comment (aremo-ms) [reply 2-WebApp-graph-user/2-3-Multi-Tenant/README.md]: one of them Overview for Scenario

**Pull request merged**